backend/src/main/models.py

```
@@ -0,0 +1,40 @@
+from pydantic import BaseModel, Field
+from typing import Optional
+
+class Setup(BaseModel):
+        MONGO_URI: str = Field(
```

Pylint (3.3.7)

[warning] Bad indentation. Found 8 spaces, expected 4

(W0311)